Tests/SWBTaskConstructionTests/SwiftTaskConstructionTests.swift

```diff
@@ -3917,6 +3917,61 @@ fileprivate struct SwiftTaskConstructionTests: CoreBasedTests {
             results.checkNoDiagnostics()
         }
     }
+    @Test(.requireSDKs(.macOS))
```

Comment:
Nothing about this test looks macOS-specific, so we should use `.requireSDKs(.host)` in order to maintain coverage across all host platforms including Window, Linux, etc.

Reply:
The existing tests were non-functional, so I removed them. I'm currently facing challenges with developer testing these changes and writing appropriate unit tests. Any guidance you can provide on the best approach, or suggestions on references within the codebase, would be greatly appreciated.

cc: @owenv @jakepetroules

Reply:
For tests I'd recommend starting out with a copy of SwiftTaskConstructionTests.upcomingFeatures(). This test is testing passing `-enable-upcoming-feature` to the Swift compiler, which is pretty similar to the new settings you're adding here. Then, I would make two changes:

1. Update the test project to use the new settings you added
2. Update the `tester.checkBuild` call at the bottom of the test to assert the Swift command line has the right flags corresponding to those settings